libqtile/scripts/qtile_top.py

```diff
@@ -158,6 +158,8 @@ def main():
     else:
         socket = opts.socket
     client = ipc.Client(socket)
+    client = command_interface.IPCCommandInterface(client)
+    client = command_client.InteractiveCommandClient(client)
```

Comment:
Can't you replace all these lines with just `client = command_client.CommandClient()`?

IIRC @pawamoy did something for that recently.

Reply:
I did in #1635, however `client = command_client.CommandClient()` will always use `find_sockfile`, which might not be wanted here.

Reply:
I think we should pester @flacjacket to write down how all this works :)

Reply:
That is a good idea :) I'll see about updating the docs. Better names for these may also be helpful, I can think about it as I write it up, but would be open if anyone has any suggestions.

This change looks good!